packages/material-ui-lab/src/useAutocomplete/useAutocomplete.js

```diff
@@ -822,7 +822,7 @@ export default function useAutocomplete(props) {
   };
 
   const handleInputMouseDown = event => {
-    if (inputValue === '') {
```

Comment:
if inputValue is empty value combobox will close.

Reply:
Yes, this is the expected behavior.

Reply:
@oliviertassinari  thank you very much for the comment.
But I don't understand 
```
  const handlePopupIndicator = event => {
    if (open) {
      handleClose(event);
    } else {
      handleOpen(event);
    }
  };
```
Why we calling handleClose() method at above.I think causes it to clicking on the GitHub input's demo is closing the popup.Can you explain that

Reply:
You can blame the line and the corresponding tests to gain more insight into the why. It should lead you to the underlying GitHub issues (that are now closed).

Reply:
ok thank you

Reply:
I get it now